examples/testprograms/grpc/autokitteh-grpc.yaml

```diff
@@ -0,0 +1,15 @@
+version: v1
+
+project:
```

Comment:
i don't think you need to suffix the autokitteh with -grpc, just drop it

Reply:
fixed